tests/foreman/ui/test_leapp_client.py

```diff
@@ -0,0 +1,77 @@
+"""Tests for leapp upgrade of content hosts with Satellite
+
+:Requirement: leapp
+
+:CaseComponent: Leappintegration
+
+:Team: Rocket
+
+:CaseImportance: Critical
+
+:CaseAutomation: Automated
+
+"""
+
+import pytest
+
+synced_repos = pytest.StashKey[dict]
+
+RHEL7_VER = '7.9'
+RHEL8_VER = '8.10'
+RHEL9_VER = '9.4'
+
+
+@pytest.mark.tier3
+@pytest.mark.parametrize(
+    'upgrade_path',
+    [
+        {'source_version': RHEL8_VER, 'target_version': RHEL9_VER},
+    ],
+    ids=lambda upgrade_path: f'{upgrade_path["source_version"]}'
+    f'_to_{upgrade_path["target_version"]}',
+)
+def test_leapp_preupgrade_report(
+    module_target_sat,
+    custom_leapp_host,
+    upgrade_path,
+    verify_target_repo_on_satellite,
+    precondition_check_upgrade_and_install_leapp_tool,
+    module_sca_manifest_org,
+):
+    """Test to verify leapp preupgrade report gets generated.
+
+    :id: cbbd38a5-65bc-4491-831a-f1bfeb119d92
+
+    :BZ: 2168494
```

Comment:
Can you also add corresponding Jira card number in the`verifies` token for customer_scenario checks and future features.

Reply:
Updated.